app/views/application_drafts/_form.html.slim

```diff
@@ -72,7 +72,15 @@ p.attention
       = f.input :voluntary_hours_per_week, as: :integer, label: 'As a voluntary team, how many hours per week would you be able to work on the project?', hint: 'Please only enter a full number, e.g. 15', input_html: { disabled: !application_draft.as_student? || application_draft.applied? }
 
       h2 Miscellaneous
-      = f.input :heard_about_it, as: :text, label: 'How have you heard about the program?', input_html: { rows: 3, disabled: !application_draft.as_student? || application_draft.applied? }
+
+      h3 How have you heard about the program?
+      = f.input :heard_about_it, collection: ApplicationDraft::DIRECT_OUTREACH_CHOICES.map {|choice| [choice, choice]}, as: :check_boxes, label: "RGSoC direct outreach", :input_html => {multiple: true }, error: false
```

Comment:
I'm not sure if I understand this correctly. So this will submit 4 values for the same key and therefore and up as an array? How do you make sure it preserves the order?